play-interop-test-scala/src/test/scala/akka/grpc/play/api/specs2/ServerGrpcClient.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.play.api.specs2
+
+import akka.grpc.internal.{AkkaGrpcClient, AkkaGrpcClientFactory}
+import akka.grpc.play.AkkaGrpcClientHelpers
+import play.api.test.{NewForServer, RunningServer}
+
+/**
+ * Helpers to test gRPC clients with Play using Specs2.
+ *
+ * Mixes a method into [[AkkaGrpcClientHelpers]] that knows how to configure
+ * gRPC clients for the running server.
+ */
+trait ServerGrpcClient extends AkkaGrpcClientHelpers { this: NewForServer =>
+
+  /** Configure the factory by combining the app and the current implicit server information */
+  implicit def configuredAkkaGrpcClientFactory[T <: AkkaGrpcClient: AkkaGrpcClientFactory](implicit server: RunningServer): AkkaGrpcClientFactory.Configured[T] = {
+    AkkaGrpcClientHelpers.factoryForAppEndpoints(app, server.endpoints)
```

Comment:
Takes a gRPC client factory and some info about the running server and creates a new factory that's configured with server information such as port number.